core/src/test/java/io/questdb/cairo/AbstractCairoTest.java

```diff
@@ -51,17 +54,17 @@
     protected static CairoConfiguration configuration;
     protected static MessageBus messageBus;
 
-    @BeforeClass
-    public static void setUp() {
-        // it is necessary to initialise logger before tests start
-        // logger doesn't relinquish memory until JVM stops
-        // which causes memory leak detector to fail should logger be
-        // created mid-test
-        LOG.info().$("begin").$();
-        root = temp.getRoot().getAbsolutePath();
-        configuration = new DefaultCairoConfiguration(root);
+	@BeforeClass
+	public static void setUp() throws IOException {
+		// it is necessary to initialise logger before tests start
+		// logger doesn't relinquish memory until JVM stops
```

Comment:
what happened here with padding?

Reply:
I need to figure out how to configure the code formatter ...